modules/op-event-pruner-configuration.adoc

```diff
@@ -0,0 +1,98 @@
+// This module is included in the following assemblies:
+// * install_config/customizing-configurations-in-the-tektonconfig-cr.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="event-pruner-configuration_{context}"]
+= Enabling the event-based pruner
+
+:FeatureName: The event-based pruner
+include::snippets/technology-preview.adoc[]
+
+You can use the event-based `tektonpruner` controller to automatically delete completed resources, such as `PipelineRuns` and `TaskRuns`, based on configurable policies. Unlike the default job-based pruner, the event-based pruner listens for resource events and prunes resources in near real time.
+
+You can use the event-based pruner alongside:
```

Comment:
I don't like the word alongside being used here. I suggest this alternative instead:

You can use the event-based pruner together with the following:

* Resource annotations
* {tekton-results}
* {PAC}